tests/sources/generic.py

```diff
@@ -56,11 +56,11 @@ async def main(queue: asyncio.Queue, args: Dict[str, Any]):
     time_format = args.get("time_format", "local")
     create_index = args.get("create_index", "")
 
-    startup_delay = int(args.get("startup_delay", 0))
-    event_delay = int(args.get("event_delay", 0))
-    repeat_delay = int(args.get("repeat_delay", 0))
-    loop_delay = int(args.get("loop_delay", 0))
-    shutdown_after = int(args.get("shutdown_after", 0))
+    startup_delay = float(args.get("startup_delay", 0))
```

Comment:
@Alex-Izquierdo Can you also fix the doc section it currently says int
https://github.com/ansible/ansible-rulebook/blob/867cb8c77a7d0614ebb0ae011b58ce0842d4ae54/tests/sources/generic.py#L33

Reply:
done